site/_layouts/post.html

```diff
@@ -0,0 +1,80 @@
+<!-- Blog Layout Start -->
+{% assign abc = page.author | split: "," %}
+{% if abc.size > 1 %}
+{% assign img_link = abc.last %}
+{% else %}
+{% assign img_link = "http://pageanthive.com/images/empty-profile.png" %}
```

Comment:
Can we use this URL? If not, then you might be [hotlinking](https://simple.wikipedia.org/wiki/Hotlinking). Best to self-host and add it to assets

Also I don't know what the permissions on that image are. You could use a very similar one which I found [here](https://pixabay.com/en/blank-profile-picture-mystery-man-973460/) which is licensed under CC0.

Reply:
Hotlinking is a very bad practice. Also can be dangerous because some sites conditionally load a different image for hotlinkers to thwart them. [famous example](https://www.theguardian.com/media/2015/oct/28/cartoonist-the-oatmeal-trolls-huffpo-over-images-published-sans-permission) 

See this [stackexchange thread](https://webmasters.stackexchange.com/questions/25315/hotlinking-what-is-it-and-why-shouldnt-people-do-it) for technical info. We shouldn't merge without addressing it.

Reply:
Cool